packages/react-icons/src/utils/fonts/createFluentFontIcon.tsx

```diff
@@ -95,5 +95,6 @@ export function createFluentFontIcon(displayName: string, codepoint: string, fon
         return <i {...state}>{codepoint}</i>
     }
     Component.displayName = displayName;
+    Component.codepoint = codepoint;
```

Comment:
To fix the build, I think you need to change the type on line 73 (and maybe also 72) to
```ts
React.FC<FluentIconsProps<React.HTMLAttributes<HTMLElement>>> & { codepoint: string}
```

Reply:
@MLoughry pinging you about this update, thanks

Reply:
I aproved, but you'll need a review with someone with write access (like @tomi-msft)